spec/models/devotional_spec.rb

```diff
@@ -49,4 +49,12 @@
       )
     end
   end
+
+  describe '.get_devotionals' do
```

Comment:
Now that we don't have that class method, do we still need this test?

Reply:
Nope! Good call!